src/main/java/org/kohsuke/github/GHEventPayload.java

```diff
@@ -1387,6 +1409,15 @@ public void setPusher(Pusher pusher) {
             this.pusher = pusher;
         }
 
+        /**
+         * Gets compare.
+         *
+         * @return compare
+         */
+        public String getCompare() {
+            return compare;
```

Comment:
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#push
Has `compare` in the example but not in the documented fields.  Not sure which is in error. 😢

Reply:
You can check the `Webhook payload example` on the link that you gave, the field is here (but not listed in description, right)